gpg_import.py

```diff
@@ -17,10 +18,23 @@
 options:
   key_id:
     description:
-      - "The id of the key to be imported"
-    required: true
+      - The id of the key to be fetched and imported. Only applicable to private keys (for now). Either key_file or key_id is required.
+    required: false
+    default: null
+
+  key_file:
+    description:
+      - Filename of key to be imported. Must be on remote machine, not local. Only applicable to public keys (for now). Either key_file or key_id is required.
+    required: false
     default: null
 
+  key_type:
+    description:
+      - What type of key to import.
+    required: true
+    choices: [ "private", "public" ]
+    default: "private"
```

Comment:
Defaulting to private key imports seems unintuitive—I'd wager most gpg-related import tasks are for public keys, e.g. for verifying signatures.